Doc/c-api/intro.rst

```diff
@@ -174,6 +174,23 @@ complete listing.
    .. versionchanged:: 3.8
       MSVC support was added.
 
+.. c:macro:: PyDoc_STRVAR(name, str)
+
+   Creates a variable called *name* that can be used in docstrings.
+
+   Use the :c:macro:`PyDoc_STRVAR` or :c:macro:`PyDoc_STR` macros for
```

Comment:
Since you mentioned `PyDoc_STR` in here, let's add documentation for it as well. Just something simple like, `Returns a docstring for the given input string`

Then the note about pep7 and example can be combined, maybe expanding the example with:
```c
...
    {"append", (PyCFunction)deque_append. METH_NOARGS, PyDoc_STR("some documentation")
...
```